kDriveCore/Data/Cache/BackgroundRealm.swift

```diff
@@ -81,9 +81,12 @@ public class BackgroundRealm {
     }
 
     public func execute(_ block: (Realm) -> Void) {
+        let activity = ExpiringActivity()
```

Comment:
I added this quick win to make sure it won't create 'deadlock crashes' since more work is performed asynchronously.

Reply:
We've had this mecanism for a long time now on Mail. I wonder why we didn't implement it earlier in kDrive